portal-test/src/com/liferay/portal/kernel/test/AssertUtils.java

```diff
@@ -127,20 +127,11 @@ public static void assertFailure(
 			Assert.fail();
 		}
 		catch (Exception exception) {
-			if (clazz != null) {
```

Comment:
Hey, could you please check if this change is related [to](https://testray.liferay.com/home/-/testray/case_results/2357586883?redirect=https%3A%2F%2Ftestray.liferay.com%2Fhome%2F-%2Ftestray%2Fcase_results%2Findex%3FtestrayBuildId%3D2357582358%26cur%3D1%26delta%3D200%26tab%3Druns%26orderByCol%3Dstatus_sortable%26orderByType%3Dasc) 

```
PACKAGE_NAME                                       DELTA      CUR_VER    BASE_VER   REC_VER    WARNINGS  
[beanshell] = ================================================== ========== ========== ========== ========== ==========
[beanshell] * com.liferay.portal.kernel.test                     MAJOR      2.4.0      2.4.0      3.0.0      VERSION INCREASE REQUIRED
[beanshell] 	>   class      com.liferay.portal.kernel.test.AssertUtils
[beanshell] 		-   method     assertFailure(java.lang.String,com.liferay.petra.function.UnsafeSupplier)
[beanshell] 			-   access     static
[beanshell] 			-   return     void
[beanshell] [Baseline Warning] Bundle Version Change Recommended: 17.0.0
```